src/Microsoft.AspNetCore.Razor.Language/RazorProjectEngineBuilderExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public static class RazorProjectEngineBuilderExtensions
+    {
```

Comment:
Should we add a convenience method for adding source documents as imports? Today you need to write your own discoverer.